store/git.go

```diff
@@ -48,7 +49,7 @@ func WriteNotes(writef func(io.Writer) error, ref string) error {
 		return fmt.Errorf("Error closing .git-ratchet-note %s", err)
 	}
 
-	writenotes := exec.Command("git", "notes", "--ref="+ref, "add", "-f", "-F", notepath)
+	writenotes := exec.Command("git", "notes", "--ref="+ref, "add", "-f", "-F", filepath.Abs(notepath))
```

Comment:
> store/git.go:52: multiple-value filepath.Abs() in single-value context

https://golang.org/pkg/path/filepath/#Abs

filepath.Abs() returns (string, error) -> you'll need to handle the potential error there before passing it through to the exec.Command.